jekyll/_cci2/language-android.md

```diff
@@ -24,128 +24,50 @@ This guide provides an introduction to Android development on CircleCI.
 If you are looking for a `.circleci/config.yml` template for Android,
 see the [Sample Configuration](#sample-configuration) section of this document.
 
-**Note:**
-We now have an Android machine image available in preview on CircleCI Cloud that supports x86 Android emulators and nested virtualization. Documentation on how to access it is available [here](https://github.com/CircleCI-Public/android-image-preview-docs).
-Another way to run emulator tests from a job is to consider using an external service like [Firebase Test Lab](https://firebase.google.com/docs/test-lab).
-For more details,
-see the [Testing With Firebase Test Lab](#testing-with-firebase-test-lab) section below.
+**Note:** We now have an Android machine image available in preview on CircleCI
+Cloud that supports x86 Android emulators and nested virtualization.
+Documentation on how to access it is available
+[here]({{site.baseurl}}/2.0/android-machine-image). Another way to run emulator
+tests from a job is to consider using an external service like [Firebase Test
+Lab](https://firebase.google.com/docs/test-lab). For more details, see the
+[Testing With Firebase Test Lab](#testing-with-firebase-test-lab) section below.
 
 ## Prerequisites
 {:.no_toc}
 
 This guide assumes the following:
 
-- You are using [Gradle](https://gradle.org/)
-to build your Android project.
-Gradle is the default build tool
+- You are using [Gradle](https://gradle.org/) to build your Android project. Gradle is the default build tool
 for projects created with [Android Studio](https://developer.android.com/studio).
 - Your project is located in the root of your VCS repository.
 - The project's application is located in a subfolder named `app`.
 
 ## Sample configuration
```

Comment:
I was thinking we could keep the current Sample configuration section, as there could be users who still prefer to use the Docker executor, especially if they are not going to run emulator-based tests? Maybe we could label it as `## Sample configuration for unit tests` , and introduce a new `Sample configuration for UI tests` section?